File: sdkwork-app-sdk-go/types/export_result_vo.go
package types

// 导出结果VO
type ExportResultVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	ExportId string `json:"exportId"`
	ReportType string `json:"reportType"`
	Status string `json:"status"`
	DownloadUrl string `json:"downloadUrl"`
	FileSize int `json:"fileSize"`
	CompletedAt string `json:"completedAt"`
	ErrorMessage string `json:"errorMessage"`
}

File: sdkwork-app-sdk-csharp/Models/ExportResultVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class ExportResultVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? ExportId { get; set; }
        public string? ReportType { get; set; }
        public string? Status { get; set; }
        public string? DownloadUrl { get; set; }
        public int? FileSize { get; set; }
        public string? CompletedAt { get; set; }
        public string? ErrorMessage { get; set; }
    }
}
